### PR TITLE
add default tags for agentops.init

### PR DESCRIPTION
**Pull request by areibman**
#91

### CODE REVIEW

File: agentstack/templates/autogen/{{cookiecutter.project_metadata.project_slug}}/src/main.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ def main():
     # For example, if you have created a OAI_CONFIG_LIST file in the current working directory, that file will be used.
     config_list = config_list_from_json(env_or_file="OAI_CONFIG_LIST")
 
-    {%- for agent in cookiecutter.structure.agents %}
```
Comment (bboynton97) [inline agentstack/templates/autogen/{{cookiecutter.project_metadata.project_slug}}/src/main.py:17]: please disable your formatter for this project. the additional whitespaces here will break cookiecutter

**Comment by bboynton97** [comment]
Adding tags for agentops is a great idea, but there are a ton of format changes in this PR that will break cookiecutter and need to be reverted

**Review by bboynton97 — CHANGES_REQUESTED** [review]
^

**Comment by areibman** [comment]
> ^

Fixed. Although it doesn't seem we have a standard linter for this, so I ended up auto formatting a bunch of the examples. Ran `ruff` to make things consistent. We should have a formatter or pre commit like in agentops

**Comment by bboynton97** [comment]
#82 - discussing it here

**Comment by bboynton97** [comment]
please remove all formatting changes, its all addressed in #82. i'd rather keep this PR just focused on resolving #92

**Comment by areibman** [comment]
> please remove all formatting changes, its all addressed in #82. i'd rather keep this PR just focused on resolving #92

Done!

**Review by bboynton97 — APPROVED** [review]

**Pull request merged**